try{
    console.log(100);
}catch(error){
    console.log(error);
}finally{
    console.log('done');
}